apps/client/src/app/pages/home-page/SocialMediaButton.tsx

```diff
@@ -0,0 +1,48 @@
+import Button from '@map-shared-components/Button';
+import FacebookLogo from '@/assets/facebook-logo.png';
+import InstagramLogo from '@/assets/instagram-logo.png';
+import TwitterLogo from '@/assets/twitter-logo.png';
+
+function SocialMediaButtons() {
+  const oneFacebookClick = () => {
+    window.location.href = 'https://es-la.facebook.com/';
+  };
+  const oneInstagramClick = () => {
+    window.location.href = 'https://www.instagram.com/';
+  };
+  const oneeTwitterClick = () => {
+    window.location.href = 'https://twitter.com/';
+  };
+
+  return (
+    <div className='container mx-auto flex justify-center  className=w-full sm:w-1/2 md:w-auto m-0.5'>
+      <div className='text-center font-roboto '>
+        <div>
+          <Button onClick={oneFacebookClick}>
```

Comment:
please these three maybe each could be a common component each could be named social media button, not buttons